modules/stochastic_tools/src/surrogates/GaussianProcessTrainer.C

```diff
@@ -44,6 +44,11 @@ GaussianProcessTrainer::validParams()
       "tao_options", "", "Command line options for PETSc/TAO hyperparameter optimization");
   params.addParam<bool>(
       "show_optimization_details", false, "Switch to show TAO or Adam solver results");
+  params.addRangeCheckedParam<Real>(
+      "prior_std",
+      1000.0,
+      "prior_std>0.0",
+      "The standard deviation of a Normal prior over the log of hyper-params");
```

Comment:
Is this option valid for TAO optimization? If not, I'd recommend throwing an error if it is specified and not used.

Reply:
I don't know the validity of this to TAO because TAO seems to formulate its own loss function. Sure, I will add an errror.